phase3/Kconfig

```diff
@@ -40,4 +40,10 @@ config phase3.weave_net
 	  Should we deploy weave-net? You likely only want this if using kubeadm for phase2.
 endif
 
+config phase3.cni
+	string "CNI plugin to use"
+	default ""
+	help
+	  CNI plugin to use. You likely only want this if using kubeadm for phase2.
```

Comment:
Since this isn't a simple yes/no boolean, can you add to the help message what the valid options are? Similar to how we do [here](https://github.com/kubernetes/kubernetes-anywhere/blob/f117f19235b5e92f9a80a4ca839352a0443195ee/phase2/Kconfig#L26).